src/sentry/testutils/helpers/data_blobs.py

```diff
@@ -0,0 +1,233 @@
+GITHUB_ACTION_DATA_BLOBS = [
+    {
```

Comment:
Not sure if this applies here, but for each distinct example I've found it helpful to label them with a name describing the contents/state it represents.